css/index.css

```diff
@@ -3,144 +3,395 @@
-    margin: 0 auto;
+  max-width: 400px;
+  margin: 0 auto;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: repeat(1fr, 5);
+}
+
+/* Header */
+
+header {
+  max-width: 400px;
+  width: 100vw;
+  height: 49vh;
+  display: grid;
+  grid-template-rows: 35% 65%;
+}
+
+nav ul {
+  display: grid;
+  height: 100%;
+  grid-template-rows: repeat(1fr, 5);
+}
+
+nav ul a {
+  display: flex;
+  align-content: center;
+  align-items: center;
+  justify-content: center;
+  border: solid #e4e4e4 1px;
+  color: black;
+  text-decoration: none;
+  width: 100%;
+  height: 100%;
+  font-size: 1.6rem;
+}
+
+header img {
+  margin: 9% 30%;
+  max-width: 162px;
+  grid-row: 1;
 }
 
+/* Top Content */
+
 .top-content {
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: space-evenly;
-    margin-bottom: 20px;
-    border-bottom: 1px dashed black;
+  display: grid;
+  grid-template-columns: 1fr;
+  margin: 0 3%;
+  border-bottom: 1px dashed black;
 }
 
 .top-content .text-container {
-    width: 48%;
-    padding: 0 1%;
-    padding-bottom: 20px;  
+  width: 100%;
+  margin: 2% 0;
+  font-size: 1.2rem;
 }
 
+.top-content img {
+  width: 106%;
+  margin: 0 -3%;
+}
+
+/* Middle Content */
+
 .middle-content {
-    margin-bottom: 20px;
-    border-bottom: 1px dashed black;
+  margin: 0 3%;
+  border-bottom: 1px dashed black;
 }
 
 .middle-content h2 {
-    padding: 0 2%;
-    margin-bottom: 0;
+  padding: 0 2%;
+  margin-bottom: 0;
 }
 
 .middle-content .boxes {
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: space-evenly;
+  display: grid;
+  grid-template-columns: 25% 25% 25%;
+  justify-content: space-around;
 }
 
 .middle-content .boxes .box {
-    width: 12.5%;
-    height: 100px;
-    background: black;
-    margin: 20px 2.5%;
-    color: white;
-    display: flex;
-    align-items: center;
-    justify-content: center;
+  width: 100%;
+  height: 100px;
+  background: black;
+  margin: 20px 0;
+  color: white;
+  display: flex;
+  align-items: center;
+  justify-content: center;
 }
 
+/* Bottom Content */
+
 .bottom-content {
-    display: flex;
-    margin: 0 2% 20px;
-    justify-content: space-around;
+  display: grid;
+  margin: 5% 2%;
+  grid-template-columns: 1fr;
+  justify-content: space-around;
 }
 
 .bottom-content .text-container {
-    padding-right: 4%;
+  padding-right: 4%;
 }
 
 .bottom-content .text-container:last-child {
-    padding-right: 0;
+  padding-right: 0;
 }
 
 footer {
-    width: 100%;
-    background: black;
+  width: 100%;
+  background: black;
 }
 
 footer nav {
-    width: 60%;
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-    padding: 20px 2%;
-    font-size: 14px;
+  width: 60%;
+  display: flex;
+  margin: 0 auto;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px 2%;
+  font-size: 1.4rem;
 }
 
 footer nav a {
-    color: white;
-    text-decoration: none;
+  color: white;
+  text-decoration: none;
+}
+
```

Comment:
For your phone-view, there is a horizontal scroll bar that seems like it is going across all the way to the tablet mode. And there are a few issues with the nav bar overlapping the header image. Also, if you take a closer look, there is no display of box10 in the phone view as well!